cypress/data/index.js

```diff
@@ -24,6 +24,11 @@ export const HIV_PROGRAM = {
     [DIMENSION_ID_LAST_UPDATED]: 'Last updated on',
 }
 
+export const TEST_AO = {
```

Comment:
rename to something that gives semantic meaning, e.g. `TEST_AO_WITH_INTERPRETATION` or something, to distinguish it from other test AOs that we'll add later for different use-cases..

Reply:
There's nothing special with the AO used for interpretations tests, so we stick to one `TEST_AO` for now.